include/arch/arm/cortex_r/scripts/linker.ld

```diff
@@ -158,11 +158,6 @@ SECTIONS
  */
 #include <snippets-text-start.ld>
 
-	/* Kinetis has to write 16 bytes at 0x400 */
-	SKIP_TO_KINETIS_FLASH_CONFIG
```

Comment:
I want someone from NXP to explain what this is supposed to do. Can't find its purpose upstream, so it shouldn't probably be upstream. In my mind, this is safe to remove like it's done.

Reply:
I think this is an earlier design which was neglected when changing it in cortex_m/*/linker.ld.

Reply:
Okei thanks for the clarification